particles/Native/Wasm/source/TestParticle.kt

```diff
@@ -91,8 +91,8 @@ class TestParticle : Particle() {
 
         eventHandler("add") {
           val newData = data.get()!!
```

Comment:
Nit: in the future consider using `requireNotNull(data.get()) { "message" }` instead of `!!` to provide a custom message when something that shouldn't be null ends up being null.